src/components/Search/Search.js

```diff
@@ -0,0 +1,49 @@
+import React, { PropTypes, Component } from 'react';
+import cx from 'classnames';
+
+import styles from './Search.scss';
+import SearchIcon from './assets/light-searchIcon.png';
+import GitHubIcon from './assets/Octocat.jpg';
+
+class Search extends Component {
+
+    static propTypes = {
+        heading: PropTypes.string.isRequired,
+        inputValue: PropTypes.string,
+        buttonText: PropTypes.string.isRequired,
+        errorMessage: PropTypes.string,
+        onSubmit: PropTypes.func,
+    };
+
+
+    render () {
+        const {heading, inputValue, buttonText, errorMessage, onSubmit} = this.props;
+      return (
+          <div>
+              <div className="row">
+                  <img src={GitHubIcon} className={styles.gitHubIcon} alt="twitter-icon" height="80" width="80"/>
+                  <div className={styles.label}>
+                      <h3>{heading}</h3>
+                  </div>
+              </div>
+              <div className="row">
+                  <div className={cx('col-xs-6')}>
```

Comment:
Can remove the cx() use here (and on the rest of these divs). Because they're just strings, you can just write className="col-xs-6" etc.

cx() is for inserting multiple classnames.